.github/workflows/ci.yml

```
@@ -58,9 +63,6 @@ jobs:
           DIST_NAME: ${{ env.dist-name }}
           VERSION: ${{ steps.dist-version.outputs.version }}
 
-      - name: Install build
-        run: python -Im pip install build
-
       - name: Build dists
         run: python -Im build
```

Assuming, you'll do something like `pip-compile --only-build-deps --output-file=build-requirements.txt pyproject.toml`:
```suggestion
        env:
          PIP_CONSTRAINT: build-requirements.txt
        run: python -Im build
```